aspnetcore/tutorials/build-your-first-blazor-app/samples/3.x/RazorComponents/README.md

```diff
@@ -0,0 +1,3 @@
+# Build Your First Blazor App Sample (RazorComponents)
+
+This sample illustrates the basic scenarios of Blazor.
```

Comment:
If this goes with a tutorial, can we add a link to the tutorial?

Reply:
Yes, I'll link it on a commit. Stand-by.